hack/parallel.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash -e
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2020 Red Hat, Inc.
+#
+
```

Comment:
@oshoval Lets add 'set -e' so we will stop on first failure.

Reply:
there is -e in the shebang
https://github.com/kubevirt/kubevirt/pull/3494/files#diff-83b5aafadb28ac3fa63881bfe735c15bR1

Reply:
@oshoval oh sorry I missed that one